Configuration/Eras/python/Era_Run3_pp_on_PbPb_approxSiStripClusters_2024_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_2024_cff import Run3_2024
+from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA 
+from Configuration.ProcessModifiers.approxSiStripClusters_cff import approxSiStripClusters
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
```

Comment:
for my education is there a reason why this is using `pp_on_PbPb_run3` as opposed to `pp_on_PbPb_run3_2024` (used in `Configuration/Eras/python/Era_Run3_pp_on_PbPb_2024_cff.py`)?